### PR TITLE
[ENH] Fixed the order of modality labels

### LINKED ISSUE #238: Ensure that the modality "labels" are ordered the same way.
Ensure that the "labels" of the the different modality on the card of each dataset are always ordered the same way.

Maybe it is my OCD but having:

- T1w, T2W, flow

and then
- flow, T2W

is a bit disturbing in terms of UX.

### PR DESCRIPTION
**Pull request by rmanaem**
Closes #238 

## Checklist

- [x] PR has an interpretable title with a prefix (`[ENH]`, `[FIX]`, `[REF]`, `[TST]`, `[CI]`, `[MNT]`, `[INF]`, `[MODEL]`, `[DOC]`) _(see https://neurobagel.org/contributing/pull_requests for more info)_
- [x] PR links to GitHub issue with mention `Closes #XXXX`
- [x] Tests pass
- [x] Checks pass

For new features:
- [x] Tests have been added

For bug fixes:
- [ ] There is at least one test that would fail under the original bug conditions.

### CODE REVIEW

File: components/ResultsContainer.vue [hunk 1]
```diff
@@ -75,7 +75,7 @@
           :dataset-uuid="res.dataset_uuid"
           :dataset-name="res.dataset_name"
           :num-matching-subjects="res.num_matching_subjects"
-          :image-modals="res.image_modals"
+          :image-modals="res.image_modals.sort()"
```
Comment (surchs) [inline components/ResultsContainer.vue]: could we have a test for this that breaks before the PR? If too tricky: please make an issue to add.

**Review by surchs — APPROVED** [review]
looks good, but test would be nice so it doesn't sneak back

**Comment by surchs** [comment]
:cook:

File: components/ResultsContainer.vue (hunk 1 above)
Comment (rmanaem) [reply components/ResultsContainer.vue]: Done

**Pull request merged**